src/app/routes/home/getInitialData/index.js

```diff
@@ -1,10 +1,15 @@
 import pipe from 'ramda/src/pipe';
+import pathOr from 'ramda/src/pathOr';
+import { getRadioScheduleEndpoint } from '#lib/utilities/getRadioSchedulesUrls';
+import serviceConfigs from '#server/utilities/serviceConfigs'; // or we could just have an array of services.
```

Comment:
Pulling in config like this massively increases bundle size, so will need to do another way